### ISSUE TITLE
Check prefixed strings

### ISSUE THREAD

**Issue by xZise** [zheller/flake8-quotes#28]
Would it be sensible to check raw strings, `unicode` or `bytes`? I'm not exactly sure what your intentions are, but in at least `u'…'` should be checked, as it is a normal string on Python 3. And the change seems reasonably simple as the token contains the prefix. You could either strip the first character if it's in `rub` or if it is not in `"'`. Any invalid prefix should result in an error anyway.

You could use different error codes so that if someone doesn't care about how the quotes are formatted in raw strings, they can just ignore that code.

**Comment by twolfson** [comment]
Ah, nice catch. I think a lack of support was unintentional. It looks like the fix should be adding a few more `startswith` checks (e.g. `b`, `br`, `u`, `UR`) :

```
>>> [x for x in tokenize.generate_tokens(lambda L=iter(['"abc"']): next(L))][(3, '"abc"', (1, 0), (1, 5), '"abc"'), (0, '', (2, 0), (2, 0), '')]
>>> [x for x in tokenize.generate_tokens(lambda L=iter(['u"abc"']): next(L))]
[(3, 'u"abc"', (1, 0), (1, 6), 'u"abc"'), (0, '', (2, 0), (2, 0), '')]
>>> [x for x in tokenize.generate_tokens(lambda L=iter(['r"abc"']): next(L))]
[(3, 'r"abc"', (1, 0), (1, 6), 'r"abc"'), (0, '', (2, 0), (2, 0), '')]
>>> [x for x in tokenize.generate_tokens(lambda L=iter(['b"abc"']): next(L))]
[(3, 'b"abc"', (1, 0), (1, 6), 'b"abc"'), (0, '', (2, 0), (2, 0), '')]
>>> [x for x in tokenize.generate_tokens(lambda L=iter(['ur"abc"']): next(L))]
[(3, 'ur"abc"', (1, 0), (1, 7), 'ur"abc"'), (0, '', (2, 0), (2, 0), '')]
```

https://github.com/python-git/python/blob/715a6e5035bb21ac49382772076ec4c630d6e960/Lib/tokenize.py#L103-L120

Would you be interested in submitting a PR to add support?

**Comment by twolfson** [comment]
This has been resolved by @xZise in #32 and released in `0.4.0`
